app/controllers/admin/merchants_controller.rb

```diff
@@ -7,5 +7,29 @@ def index
   def show
     @merchant = Merchant.find(params[:id])
   end
+  
+  def edit 
+    @merchant = Merchant.find(params[:id])
+  end
+  
+  def update 
+    # require 'pry'; binding.pry
```

Comment:
Same thing as the test lets just delete these commented out pry's before pushing up PR's moving forward.